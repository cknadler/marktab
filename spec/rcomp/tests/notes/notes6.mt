# notes shortand with mutes, multiple strings
1:1 x 2 2:3 x 4
