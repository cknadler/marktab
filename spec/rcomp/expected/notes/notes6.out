e|-1-x-2-------|
B|-------3-x-4-|
G|-------------|
D|-------------|
A|-------------|
E|-------------|

